src/components/Header/Header.tsx

```diff
@@ -0,0 +1,78 @@
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos'
+import { useHistory } from 'react-router'
+import styled from 'styled-components'
+import Button from '../common/Button/Button'
+
+interface Props {
+  withBack?: Boolean
+  children: string
+}
+
+function Header({ withBack = false, children }: Props): JSX.Element {
+  const history = useHistory()
+
+  function goBack(): void {
+    history.goBack()
+  }
+
+  return (
+    <Container>
+      {withBack && (
+        <StrippedButton onClick={goBack}>
+          <ArrowBackIosIcon /> back
+        </StrippedButton>
+      )}
+      <StyledTitle>{children}</StyledTitle>
+    </Container>
+  )
+}
+
+export default Header
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(#5c5c5c33, transparent);
+    content: '';
+    height: 4px;
+    left: 0;
+    position: absolute;
+    top: 100%;
+    width: 100%;
+  }
+`
+const StrippedButton = styled(Button)`
+  align-items: center;
+  background-color: transparent;
+  border-radius: 0;
+  border: none;
+  color: var(--color-text);
+  cursor: pointer;
+  display: flex;
+  font-size: 1.5rem;
+  left: 5%;
```

Comment:
Percentage for spacing is dangerous (could get too big or too small or just look weird).